src/Container.js

```diff
@@ -20,15 +21,29 @@ export default class Container extends React.Component {
       perRequest: 100,
       limitParam: 'perPage',
       params: getSASParams({
-        searchKey: 'name',
+        searchKey: 'name,alternateNames.name,description',
         filterKeys: {
```

Comment:
I think renewalPriority filterKey is missing here and hence all the corresponding tests are failing